docs/en/attributi_utente.rst

```diff
@@ -226,41 +198,47 @@ User attributes
         }
 
      - |spid-icon| |cieid-icon|
-   * - **phone_number**
-     - extra registry
+   * - **phone_number** |br| Category: extra registry
      - Mobile phone name. String. Numeric string without internal spaces |br|
        Example: |br|
        ``"phone_number": "VATIT-12345678901"``
      - |spid-icon| |cieid-icon|
-   * - **email**
-     - extra registry
+   * - **phone_number_verified** |br| Category: extra registry
+     - Boolean value indicating whether the user's mobile phone number has been verified by the OP. 
+     - |cieid-icon|
+   * - **$PREFIX/landline_number** |br| Category: extra registry
+     - Landline number. String. Numeric string without internal spaces |br|
+       Example: |br|
+       ``"$PREFIX/landline_number":"VATIT-12345678901"``
+     - |cieid-icon|
+   * - **email** |br| Category: extra registry
      - E-mail address. String. Standard e-mail address |br|
        Example: |br|
-       ``"email": "name@domain.it"``
+       ``"email":"name@domain.it"``
      - |spid-icon| |cieid-icon|
-   * - **https://attributes.eid.gov.it/e_delivery_service**
-     - extra registry
-     - Digital domicile. PEC e-mail address |br|
-       Example: |br|
-       ``"https://attributes.eid.gov.it/e_delivery_service":"name@pecdomain.it"``
+   * - **email_verified** |br| Category: extra registry
+     - Boolean value indicating whether the user's e-mail has been verified by the OP.
      - |cieid-icon|
-   * - **https://attributes.eid.gov.it/eid_exp_date**
-     - extra registry
+   * - **$PREFIX/e_delivery_service** |br| Category: extra registry
+     - Qualified electronic registered delivery. PEC e-mail address |br|
+       Example: |br|
+       ``"$PREFIX/e_delivery_service":"name@pecdomain.it"``
+     - |spid-icon| |cieid-icon|
+   * - **$PREFIX/eid_exp_date** |br| Category: extra registry
```

Comment:
$PREFIX/id_exp_date
@nunzionapoli please fix on https://www.agid.gov.it/sites/default/files/repository_files/tabella_attributi_v.1.3.pdf

Reply:
risolto